client/ayon_blender/plugins/publish/extract_fbx_animation.py

```diff
@@ -132,13 +133,35 @@ def process(self, instance):
         max_frame = min(starting_frames)
         min_frame = max(ending_frames)
 
-        # We bake the copy of the current action for each object
-        bpy_extras.anim_utils.bake_action_objects(
-            object_action_pairs,
-            frames=range(int(min_frame), int(max_frame)),
-            do_object=False,
-            do_clean=False
-        )
+        ver_major, _, _ = get_blender_version()
+        if ver_major >= 4:
+            # We bake the copy of the current action for each object
+            bake_options = bpy_extras.anim_utils.BakeOptions(
+                only_selected=True,
+                do_pose=True,
+                do_object=False,
+                do_visual_keying=True,
+                do_constraint_clear=True,
+                do_parents_clear=True,
```

Comment:
In the old way the defaults for these were `False` and we were not passing them. As such, this seems to be 'new' behavior if we would enable them now. Should these remain disabled?

```suggestion
                do_constraint_clear=False,
                do_parents_clear=False,
```